### DISCUSSION TITLE
[✨] Multiple Flutter Versions in one System

### DISCUSSION THREAD

**Discussion by ziyadvf** [FlutterMatic/desktop#52] [Ideas]
**Is your feature request related to a problem? Please describe.**
Allow storing more than one Flutter channel on the system

**Describe the solution you'd like**
Store multiple Flutter channels at once.

**Describe alternatives you've considered**
Manually switching and basically reinstalling Flutter by checking out the current channel branch and checking in the new branch (takes time).

**Additional context**
- Storing multiple should be optional (default channel stable)
- This is planned to be available at a stable release directly (skipping alpha and beta) if enough upvotes.

**Comment by yahu1031** [comment]
This was already an existing application called [FVM(CLI)](https://fvm.app/). This will not be a good idea to overcomplicate the things WRT FVM. No one will use it cause everything with FVM is well aligned and assigned. So I'm not upvoting this feature.

**Comment by ziyadvf** [comment]
Yes, we want to integrate this into FM to minimize the use of multiple tools. This makes it easier to switch between channels and so forth. Perhaps we could integrate this tool into FM.

**Comment by yahu1031** [comment]
Nope @ZiyadF296. FVM is not so hard to use. Actually, it is very lightweight and easy as other CLI apps. And it a not a useful thing. I can't say it is useless too but it is not useful. Please make sure to add a feature to a TODO task list when contributors are on the same ship. And if the feature is accepted then add an enhancement label to it. And add it to the TODO list of the next releases. Do not waste your time on doing that which is not at all required as of now.

**Comment by yahu1031** [comment]
I'm not upvoting it 😃 👎🏻

**Comment by ziyadvf** [comment]
I created this issue as a way to collect opinions. Your opinion is appreciated :)
